src/main/java/com/ocient/jdbc/XGConnection.java

```diff
@@ -2881,9 +2900,11 @@ public void forceExternal(final boolean force) throws SQLException
 		innerBuilder.setIsOn(force);
 		builder.setForceExternal(innerBuilder.build());
 		builder.setReset(false);
-		sendParameterMessage(builder.build());	
+		CompletionStage<SQLResult<Integer>> completionStage = sendParameterMessage(builder.build());	
 		// Change this only if sendParameterMessage succeeded. It would have thrown otherwise.
 		forceExternal = force;
+
+		return completionStage.thenApply(respOrEx -> respOrEx.isException() ? SQLResult.fail(respOrEx.getException()) : SQLResult.voidResult());
```

Comment:
(nit)

This is a good use case for `SQLResult#mapSuccess`, converting `SQLResult<T>` => `SQLResult<U>` where the exception type remains the same. The `Functions::noop` method reference is just syntactic sugar.
```suggestion
		return completionStage.thenApply(respOrEx -> respOrEx.mapSuccess(Functions::noop));
```